internal/pkg/state/registry/registry.go

```diff
@@ -150,11 +154,19 @@ func (s *Registry) Configs() (configs []*ConfigState) {
 	return configs
 }
 
-func (s *Registry) IgnoreConfig(ignoreID string) {
+func (s *Registry) IgnoreConfig(ignoreID string, componentID string) {
 	for _, object := range s.All() {
 		if v, ok := object.(*ConfigState); ok {
-			if v.ID.String() == ignoreID {
+			if v.ID.String() == ignoreID && v.ComponentID.String() == componentID {
+				// ignore configuration
 				v.Ignore = true
+
+				// ignore rows of the configuration
+				if len(s.ConfigRowsFrom(v.ConfigKey)) > 0 {
```

Comment:
This condition seems useless?

Reply:
The configuration can have rows configuration. If you want to remove this configuration you need to remove rows too.

Reply:
but you are right, this condition does not have to be there

Reply:
[ae0cdbe](https://github.com/keboola/keboola-as-code/pull/2136/commits/ae0cdbebc3d40eb6f5c644e1c3c260a1c5d56030)